yb-voyager/cmd/common.go

```diff
@@ -1269,6 +1270,148 @@ func (ar *AssessmentReport) GetClusterSizingRecommendation() string {
 		ar.Sizing.SizingRecommendation.MemoryPerInstance, ar.Sizing.SizingRecommendation.EstimatedTimeInMinForImport)
 }
 
+func (ar *AssessmentReport) GetTotalTableRowCount() int64 {
+	if ar.TableIndexStats == nil {
+		return -1
+	}
+
+	var totalTableRowCount int64
+	for _, stat := range *ar.TableIndexStats {
+		totalTableRowCount += utils.SafeDereferenceInt64(stat.RowCount)
```

Comment:
we should add row counts of table only,  so should do under the check `!stat.IsIndex`

Reply:
Not sure. need to check